aws_cloudwatch/README.md

```diff
@@ -0,0 +1,40 @@
+# AWS Cloudwatch Monitoring Template
+
+Provided by: [bonitoo.io](.)
+
+**Display data from AWS EC2 andn ELB using the AWS Cloudwatch Service.**
+
+1. The AWS Cloudwatch Network Load Balancers monitoring dashboard displays data from `cloudwatch_aws_network_elb` measurement.
+2. The AWS Cloudwatch Instance Monitoring dashboard displays data from the `cloudwatch_aws_ec2` measurement.
```

Comment:
The AWS ~Cloudwatch~ **CloudWatch** Instance Monitoring dashboard displays data from the `cloudwatch_aws_ec2` measurement.